x-pack/plugins/maps/server/routes.ts

```diff
@@ -21,44 +21,40 @@ export async function initRoutes(coreSetup: CoreSetup, logger: Logger): Promise<
   const [coreStart, { data: dataPlugin }]: [CoreStart, StartDeps] =
     (await coreSetup.getStartServices()) as unknown as [CoreStart, StartDeps];
 
-  router.versioned
-    .get({
+  router.get(
```

Comment:
@nreese I am not very familiar with how MapBox treats `glyphs` endpoint and whether an additional header can be passed?

If we can send a custom header I'm happy to revert this change, just seemed safest for now to keep this endpoint unversioned.

Reply:
Maps uses Maplibre (which is a fork of mapbox when they changed there license for version 2).

Maplibre has a hook to add enable adding additional headers and this logic can be found here
https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/connected_components/mb_map/transform_request.ts#L21

Reply:
Excellent, thank you!